xmodule/js/src/sequence/display.js

```diff
@@ -62,23 +71,65 @@
             this.showCompletion = this.el.data('show-completion');
             this.keydownHandler($(element).find('#sequence-list .tab'));
             this.base_page_title = ($('title').data('base-title') || '').trim();
+            this.dropdownButtonTpl = _.template($('#dropdown-button-tpl').text())({});
+            this.renderDropdown();
             this.bind();
             this.render(parseInt(this.el.data('position'), 10));
         }
 
+        Sequence.prototype.renderDropdown = function() {
+          this.$(`#sequence-list > li.sequence-list-item`).show();
+          const tabListWidth = this.$('#sequence-list').width()
+          const singleTabWidth = this.$('#sequence-list > li:first').width()
+          // Reduce 1 to offsets index and another one to accommodate the button
+          const overFlowIdx = Math.floor(tabListWidth / singleTabWidth) - 2;
+          if (this.$('#sequence-list > #dropdown-container').length===0) {
+            this.$('#sequence-list > li.sequence-list-item').eq(overFlowIdx - 1).after(this.dropdownButtonTpl);
+          } else {
+            this.$('#sequence-list > li.sequence-list-item').eq(overFlowIdx - 1).after(this.$('#sequence-list > #dropdown-container'));
+          }
+          this.$(`#sequence-list > li.sequence-list-item:lt(${overFlowIdx})`).show();
+          this.$(`#sequence-list > li.sequence-list-item:gt(${overFlowIdx-1})`).hide();
+          const dropdownList = this.$('#dropdown-sequence-list > ol');
+          dropdownList.empty();
+          this.$(`#sequence-list > li:gt(${overFlowIdx})`).each(function(idx, el) {
+            const cloneEl = $(el).clone();
+            const navButton = cloneEl.find("button");
+            const unitTitle = navButton.data('page-title');
+            navButton.click(this.goto);
+            navButton.find(".sequence-tooltip").remove();
+            navButton.find("span.icon").after('<span class="unit-title">' + unitTitle + '</span>');
+            cloneEl.show().appendTo(dropdownList);
+          });
```

Comment:
```suggestion
          });
        this.$('#dropdown-sequence-list').hide();
```

Without this, the page always renders with the dropdown open.

Reply:
Sorry I kept it open for quicker testing during dev. I think a better approach is to set `display: none` that way it won't even flash open. I'll go back to doing that since it's closer to done .